naima/models.py

```diff
@@ -431,3 +431,76 @@ def __call__(self, e):
         e = _validate_ene(e)
         interpy = np.power(10, self._interplogy(np.log10(e.to('eV').value)))
         return self.amplitude * interpy * self.unit
+
+
+class EblAbsorptionModel(TableModel):
+    """
+    A TableModel containing the different absorption values from a specific model.
+
+    It returns dimensionless opacity values, that could be multiplied to any model.
+
+    Parameters
+    ----------
+    redshift : float
+        Redshift considered for the absorption evaluation.
+    ebl_absorption_model : string
+        Name of the EBL absorption model to use (Dominguez by default).
+
+    See Also
+    --------
+    TableModel
+
+    Notes
+    -----
+    Dominguez model refers to the Dominguez 2011 EBL model. Current implementation 
+    does NOT perform an interpolation in the redshift, so it just uses the closest
+    z value from the finely binned tau_dominguez11.out file (delta_z=0.01).
```

Comment:
the filename is now `npz`, probably should change that here so somebody looking for the file in the source can find it.